### PR TITLE
Align installation procedure with ocm repo

### PR DESCRIPTION
**Pull request by morri-son**
align installtion proceduire with ocm repo

### CODE REVIEW

File: content/docs/getting-started/installation.md [hunk 1]
```diff
@@ -10,21 +10,94 @@ weight: 21
...
+
+[AUR Documentation](https://wiki.archlinux.org/title/Arch_User_Repository)
+
+## Install using Docker / Podman
+
+```sh
+podman run -t ghcr.io/open-component-model/ocm:latest --help
+```
+
+## Build and run it yourself
```
Comment (n-boshnakov) [inline content/docs/getting-started/installation.md:77]:
```suggestion
## Build and Run It Yourself
```

File: content/docs/getting-started/installation.md [hunk 2]
```diff
@@ -10,21 +10,94 @@ weight: 21
 toc: true
 ---
 
-## Using Binaries
+Install the latest release from any of
```
Comment (n-boshnakov) [inline content/docs/getting-started/installation.md:13]:
```suggestion
## Overview 

You can install the latest release of the OCM CLI from any of of the following sources (more details below):
```

File: content/docs/getting-started/installation.md [hunk 3]
```diff
@@ -10,21 +10,94 @@ weight: 21
...
+- [AUR](https://aur.archlinux.org/packages/ocm-cli)
+- [Docker](https://www.docker.com/)
+- [Podman](https://podman.io/)
+- [GitHub Releases](https://github.com/open-component-model/ocm/releases)
 
-## Homebrew
+## Bash
 
-You can also install via [homebrew](https://brew.sh/) for macOS and Linux:
+To install with `bash` for macOS or Linux execute the following command:
```
Comment (n-boshnakov) [inline content/docs/getting-started/installation.md:24]:
```suggestion
To install with `bash` for macOS or Linux, execute the following command:
```

**Review by n-boshnakov — CHANGES_REQUESTED** [review]

**Review by frewilhelm — CHANGES_REQUESTED** [review]

**Review by frewilhelm — APPROVED** [review]

**Review by n-boshnakov — APPROVED** [review]
/lgtm

**Pull request merged**